auth-server/src/utils/getTokenFromReq.ts

```diff
@@ -9,13 +10,13 @@ export default (req: Request): string => {
 	const authHeader = req.headers.authorization
 
 	if (!authHeader) {
-		throw new Error('Authorization header missing')
+		throw new ForbiddenError('Authorization header missing')
```

Comment:
Shouldn't this rather be a `AuthenticationError`?

Reply:
done